test/unit/Query/BoolQuery.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+
+namespace Test\Unit\Novaway\ElasticsearchClient\Query;
+
+
+use atoum\test;
+use Novaway\ElasticsearchClient\Query\CombiningFactor;
+
+class BoolQuery extends test
+{
+    public function testAddQuery()
+    {
+        $mockFirstNameQuery = new \mock\Novaway\ElasticsearchClient\Query\MatchQuery('firstname', 'bruce', CombiningFactor::MUST);
```

Comment:
IMO you don't need to mock `MatchQuery` object as it's a VO and doesn't contain logic